ompi/communicator/comm.c

```diff
@@ -1331,6 +1331,17 @@ int ompi_comm_create_from_group (ompi_group_t *group, const char *tag, opal_info
 
     newcomp->instance = group->grp_instance;
 
+    /*
+     * setup predefined keyvals - see MPI Standard for predefined keyvals cached on 
+     * communicators created via MPI_Comm_from_group or MPI_Intercomm_create_from_groups
+     */
```

Comment:
Why do you cache a constant value into each communicator ? This forces you to create and initialize the keyhash, add room for few elements, and then add the UB key. Seems like a waste of memory

Reply:
I guess we could have an attr lookaside thing on communicators created via mpi_comm_from_group/mpi_intercomm_from_groups.  not sure i like doing some kind of bypass just for a modest memory savings.